pkg/pointerdb/config.go

```diff
@@ -26,6 +27,7 @@ type Config struct {
 	DatabaseURL          string `help:"the database connection string to use" default:"bolt://$CONFDIR/pointerdb.db"`
 	MinInlineSegmentSize int64  `default:"1240" help:"minimum inline segment size"`
 	MaxInlineSegmentSize int    `default:"8000" help:"maximum inline segment size"`
+	Overlay							 bool `default:"false" help:"toggle flag if overlay is enabled"`
```

Comment:
alignment

Reply:
This wasn't displaying on my machine, I think it was a tab issue. I think I fixed it now, though.

Reply:
Use gofmt/goimports on save.